__tests__/app.test.js

```diff
@@ -308,3 +308,31 @@ describe("/api/comments/:comment_id", () => {
     });
   });
 });
+
+describe("/api/users", () => {
+  describe("GET", () => {
+    test("GET 200: should return an array of user objects", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then((response) => {
+          expect(response.body.users).toBeInstanceOf(Array);
+          expect(response.body.users.length).toBeGreaterThan(0);
+          response.body.users.forEach((user) => {
+            expect(user).toHaveProperty("username");
+            expect(user).toHaveProperty("name");
+            expect(user).toHaveProperty("avatar_url");
+          });
+        });
+    });
+
+    test("GET 404: should return status code 404 for invalid path for endpoint", () => {
+      return request(app)
+        .get("/api/nonsense")
+        .expect(404)
+        .then((response) => {
+          expect(response.body.msg).toBe("Endpoint Invalid/Not Found");
+        });
+    });
+  });
```

Comment:
You don't need this test for every endpoint, once you've covered it once in your server you're good